cloud/aws/cli/plugins/misp.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
Would it make sense to include a logging mechanism to console and/or a file for implicit errors? In the issue with `misp.tunnel` I unfortunately do not directly get an error message.

Reply:
what to do mean by "logging mechanism"? Usually a CLI just prints everything that can be useful to the user to the console. Are you looking for something like a `--verbose` option? That could be an interesting addition. 

There is a way to display stack traces that is not documented in the high level docs as it is intended for development of the CLI itself, you have to set VASTCLOUD_TRACE=1 in the environment.

Reply:
I just checked the behavior and satisfied with overall verbosity on status/errors.
![image](https://user-images.githubusercontent.com/5330275/190359976-e2605264-0086-4410-a97e-84d5aa62d406.png)

Putting this to resolved.